src/_includes/partials/breadcrumbs.njk

```diff
@@ -18,18 +66,18 @@
 			{% if tags[0] != "main" %}
 
 				{% if subject %}
-					<li><a href="/{{ locale }}/{{ tagList.tags[locale][subject] | stripTagsSlugify or tagList.subjects[locale][subject] | stripTagsSlugify }}">{{ tagList.tags[locale][subject] or tagList.subjects[locale][subject] }}</a></li>
+					{% for breadcrumb in subject %}
+						<li><a href="/{{ locale }}/{{ tagList.tags[locale][breadcrumb] | stripTagsSlugify or tagList.subjects[locale][breadcrumb] | stripTagsSlugify }}">{{ tagList.tags[locale][breadcrumb] or tagList.subjects[locale][breadcrumb] }}</a></li>
+					{% endfor %}
 				{% endif %}
 
 				{% if tags %}
 					{% for breadcrumb in tags %}
-
-					{% if breadcrumb != "updatesMain" %}
-					<li>
-						<a href="/{{ locale }}/{{ tagList.tags[locale][breadcrumb] | stripTagsSlugify or tagList.subjects[locale][breadcrumb] | stripTagsSlugify }}">{{ tagList.tags[locale][breadcrumb] or tagList.subjects[locale][breadcrumb] }}</a>
-					</li>
-					{% endif %}
-
+						{% if breadcrumb != "updatesMain" %}
+							<li>
+								<a href="/{{ locale }}/{{ tagList.tags[locale][breadcrumb] | stripTagsSlugify or tagList.subjects[locale][breadcrumb] | stripTagsSlugify }}">{{ tagList.tags[locale][breadcrumb] or tagList.subjects[locale][breadcrumb] }}</a>
+							</li>
```

Comment:
Having the tags rendered as a hierarchical way may not optimal for the breadcrumb. 

I would like to suggest to present those tags differently, like a flatten list at the top or at the bottom of the current page in the main content area.

Reply:
@duboisp can you explain more what you mean by your comment?

Reply:
the "tags" are the way eleventy groups collections. We are looking to change it to the [Navigation — Eleventy plugin](https://www.11ty.dev/docs/plugins/navigation/) instead of "tags" and [Collections — Eleventy](https://www.11ty.dev/docs/collections/).

Reply:
As discussed, it is just to ensure from the breadcrumb include perspective that the `subject` array is different of the `tags` array. Currently both are behaving the same base on that logic.

For example, I am expecting that something different will happen for each those 3 variations in the front-matter, which is currently not the case with the propose logic:

```
tags:
  - item1
  - item2
  - item3
  - item4
  - item5
```

```
subject:
  - item1
  - item2
  - item3
  - item4
  - item5
```

```
subject:
  - item1
  - item2
tags:
  - item3
  - item4
  - item5
```

Reply:
Did exactly that. Thanks @duboisp for your help on this.